.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit $1
```

Comment:
How about we make use of the locally installed `commitlint` from the `node_modules` instead of relying on `npx`?

Reply:
Using `npx --no-install` will not download `commitlint`, and also will directly find and execute the locally installed `commitlint` inside `node_modules/.bin`. So both approaches are practically the same, but there is a benefit to the `npx --no-install` method:

Error message: It will show an error message to the developer, hinting them that they may be missing running `yarn` to install dependencies, whereas running it directly may display a more obscure one (ex. something along the lines of `the file does not exist`).